layouts/partials/card.html

```diff
@@ -1,19 +1,3 @@
-{{- $opts := dict
-  "page" .
-  "title" .Params.title
-  "icon" .Params.icon
-  "color" .Params.color
-  "inner" .Inner
-}}
-
-{{- with .Parent }}
-  {{- partial "inline/card.html" $opts }}
-{{- else }}
-  <div class="card-nav d-flex flex-column flex-sm-row">
-    {{- partial "inline/card.html" $opts }}
-  </div>
-{{- end }}
-
```

Comment:
I don't think we should remove this; we may wish to add cards in other places.

Secondly, I'm reasonably sure that this initial dict is to provide default values for options that are not given, so removing *that* may break things / harder to use.